core/datainput.gms

```diff
@@ -1061,7 +1068,7 @@ $elseif not "%cm_adj_coeff%" == "off"
   p_adj_coeff(t,regi,te)$p_new_adj_coeff(te) = p_new_adj_coeff(te);
 $endif
 
-p_adj_coeff(ttot,regi,te)            = 25 * p_adj_coeff(ttot,regi,te);  !! Rescaling all adjustment cost coefficients
```

Comment:
why for all? what is the meaning of 25/32?

Reply:
The general perception that REMIND is a bit too flexible in its upscaling.
So the commit 
a) slows down PV, wind, wind offshore
b) treats all <x>2h2, <x>2liquids similar (with a gas faster than coal faster than biomass due to complexity of the plant due to material handling and preprocessing) - before there were simply techs missing in the list
c) makes everything 28% higher